### PR TITLE
CB-72: Delete saved cohort

### PR DESCRIPTION
**Pull request by femioladeji**
# JIRA TICKET NAME:
[CB-72: Delete saved cohort](https://issues.openmrs.org/browse/CB-72)
## SUMMARY:
Users should be able to delete a saved cohort and all the patients associated in that saved cohort. This will require making a DELETE request to /cohort/{uuid}

### CODE REVIEW

File: app/js/components/cohorts/actionsComponent.jsx [hunk 1]
```diff
@@ -182,6 +198,8 @@ class ActionsComponent extends Component {
                                     <td>{index + 1}</td>
                                     <td>{ cohort.display }</td>
                                     <td>{cohort.description}</td>
+                                    <td><span className="glyphicon glyphicon glyphicon-remove remove" onClick={this.deleteCohort(cohort.uuid)} title="Delete" aria-hidden="true" />
```
Comment (andela-aolaniran) [inline app/js/components/cohorts/actionsComponent.jsx]: glyphicon is repeated twice.

Comment (femioladeji) [reply app/js/components/cohorts/actionsComponent.jsx]: Nice catch

File: app/js/components/cohorts/actionsComponent.jsx [hunk 2]
```diff
@@ -125,11 +130,21 @@ class ActionsComponent extends Component {
 		};
 	}
 
+    deleteCohort(cohortId) {
+        return (e) => {
+            const apiHelper = new ApiHelper();
+            apiHelper.delete(`/cohort/${cohortId}`)
+                .then(res => {
+                    this.getAllCohorts();
```
Comment (andela-aolaniran) [inline app/js/components/cohorts/actionsComponent.jsx]: I feel if we aren't using the 'res' argument returned after delete, there might be no need to include it. What do you think about this @andela-foladeji ?

Comment (femioladeji) [reply app/js/components/cohorts/actionsComponent.jsx]: True; for now, we're not using it but I feel there may be a need to use it in the future. If we want to display a form of confirmation later on then we may need to check some properties of `res` to determine the right information to display

Comment (andela-aolaniran) [reply app/js/components/cohorts/actionsComponent.jsx]: 👍

Comment (dkayiwa) [reply app/js/components/cohorts/actionsComponent.jsx]: @andela-foladeji for most of the times, those things which you think you may need in future, you never actually get to need them. So the advice here is to assume you will never need them and use only what is for today.
That way, you do not fall in the trap of maintaining code which you may never.

Comment (femioladeji) [reply app/js/components/cohorts/actionsComponent.jsx]: I agree, I'll take it off then

**Comment by femioladeji** [comment]
@dkayiwa I've made the changes

**Pull request merged**